src/components/NavBar/Buttons.js

```diff
@@ -0,0 +1,210 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import Menu from "../Icons/Menu";
+import Search from "../Icons/Search";
+import User from "../Icons/User";
+import colors from "../../theme/colors";
+import spacing from "../../theme/spacing";
+import typography from "../../theme/typography";
+import { mediumAndUp, largeAndUp } from "../../theme/mediaQueries";
+import { Consumer } from "./Context";
+import getRelByTarget from "../../utils/link";
+
+const BaseButton = styled.button`
+  display: flex;
+  align-items: center;
+  flex: 0 1 auto;
+  background-color: transparent;
+  text-decoration: none;
+  border: 0;
+  height: 60px;
+  padding-right: ${({ isLast }) =>
```

Comment:
Any reason why you are opting to use props instead of pseudo selectors for the first/last elements?

Reply:
it's not possible to always target an element with pseudo selectors because of the responsive visibility.